build_tools/sphinx/github_link.py

```diff
@@ -44,7 +44,13 @@ def _linkcode_resolve(domain, info, package, url_fmt, revision):
         # Python 2 only
         class_name = class_name.encode('utf-8')
     module = __import__(info['module'], fromlist=[class_name])
-    obj = attrgetter(info['fullname'])(module)
+
+    try:
+        obj = attrgetter(info['fullname'])(module)
+    except AttributeError as exc:
+        # print(module, attrgetter(info['fullname']))
```

Comment:
when does this happen? and do we need the debug code here?

Reply:
`Dataset.file_meta` seems to trigger it. I can remove the debug code.

The script only adds the [source] link at the top of the API page for each object anyway, and that works so I wasn't too concerned about digging into it further.